src/CMakeLists.txt

```diff
@@ -308,6 +354,9 @@ sundemo_add_executable(hurricane_yz.exe
   COMPILE_DEFINITIONS
     TEST_YZ)
 
+add_test(hurricane_yz
```

Comment:
Why was `hurricane_xy` disabled, but both `hurricane_yz` and `hurricane_zx` were enabled?  I think it's fine if none of these are included in the testing.

Reply:
`hurricane_xy` takes significantly longer (about 10x more) than the `yz` and `zx` tests so I excluded it for now but left the others/